plugins/modules/aci_domain_to_vlan_pool.py

```diff
@@ -330,21 +330,22 @@ def main():
             module_object=domain_mo,
             target_filter={"name": domain},
         ),
-        child_classes=["infraRsVlanNs"],
+        subclass_1=dict(
+            aci_class="infraRsVlanNs",
+            aci_rn="rsvlanNs",
+            target_filter={"dn": "{0}/rsvlanNs".format(domain_mo)},
+        ),
     )
 
     aci.get_existing()
 
     if state == "present":
         aci.payload(
-            aci_class=domain_class,
-            class_config=dict(name=domain),
-            child_configs=[
-                {"infraRsVlanNs": {"attributes": {"tDn": aci_mo}}},
-            ],
+            aci_class="infraRsVlanNs",
+            class_config=dict(tDn=aci_mo),
         )
 
-        aci.get_diff(aci_class=domain_class)
+        aci.get_diff(aci_class="infraRsVlanNs")
```

Comment:
Can you also update the tests in https://github.com/CiscoDevNet/ansible-aci/blob/master/tests/integration/targets/aci_domain_to_vlan_pool/tasks/main.yml?

Reply:
Test updated. But i currently dont have a local APIC to test, so if you could run the suite against your pipeline, that would be great (maybe i have some typo somewhere).
I see that the pipeline is timing out right now.

Reply:
Tests updated:
```
PLAY RECAP *********************************************************************
azure_cloud                : ok=7    changed=0    unreachable=0    failed=0    skipped=36   rescued=0    ignored=0
cn-dmz-apic-m1-02-v42      : ok=39   changed=10   unreachable=0    failed=0    skipped=4    rescued=0    ignored=1
cn-dmz-apic-m1-03-v52      : ok=39   changed=10   unreachable=0    failed=0    skipped=4    rescued=0    ignored=1
cn-dmz-apic-m1-04-v602h    : ok=39   changed=10   unreachable=0    failed=0    skipped=4    rescued=0    ignored=1
```